chapter02/dtj/ex5

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/perl
+
+use warnings;
+
+print "Enter the string: ";
+$string = <STDIN>;
+print "Enter the number of repeats: ";
+chomp($num = <STDIN>);
+print $string x $num;
```

Comment:
Correct, my bad :pensive:. Only a small thing though, if an empty string is entered (which will have only "\n"), then only new lines get printed $num times. :+1: